Library/Formula/tesseract.rb

```diff
@@ -17,6 +17,7 @@ class Tesseract < Formula
     depends_on "automake" => :build
     depends_on "libtool" => :build
     depends_on "pkg-config" => :build
+    depends_on "cairo" => :build
```

Comment:
There appears to be a change in Tesseract HEAD somewhere along the way (since 3.04.00) that introduced a warning about Cairo during the default `./configure` step (I believe it's only used for training-tools though).